client/reader/following-edit/index.jsx

```diff
@@ -34,7 +34,7 @@ import FollowingImportButton from './import-button';
 import FeedDisplayHelper from 'reader/lib/feed-display-helper';
 import SectionHeader from 'components/section-header';
 import Button from 'components/button';
-const stats = require( 'reader/stats' );
+import * as stats from 'reader/stats';
```

Comment:
Does the codemod do this? Can we determine what functions are used from these modules? `import *` isn't good for tree-shaking.